src/infra/main/template.yaml

```diff
@@ -900,32 +964,16 @@ Resources:
           Value: !Ref OwnerTagValue
         - Key: Source
           Value: !Ref SourceTagValue
+        - Key: Name
+          Value: !Sub "/aws/lambda/${InvokePrivateAPIGatewayFunction}"
 
-  StatusRetrieverFunctionPolicy:
-    Type: AWS::IAM::Policy
+  InvokePrivateAPIGatewayFunctionLogsSubscriptionFilterCSLS:
+    Condition: IsProdOrInt
+    Type: AWS::Logs::SubscriptionFilter
     Properties:
-      PolicyName: !Sub "${AWS::StackName}-StatusRetrieverFunctionPolicy"
-      PolicyDocument:
-        Version: "2012-10-17"
-        Statement:
-          - Effect: Allow
-            Action:
-              - "logs:CreateLogStream"
-              - "logs:PutLogEvents"
-            Resource:
-              - !GetAtt StatusRetrieverFunctionLogGroup.Arn
-          - Effect: Allow
-            Action:
-              - "dynamodb:Query"
-            Resource:
-              - Fn::ImportValue: !Sub "${CoreStackName}-AccountStatusTableArn"
-          - Effect: Allow
-            Action:
-              - "kms:decrypt"
-            Resource:
-              - Fn::ImportValue: !Sub "${CoreStackName}-AccountStatusTableEncryptionKeyArn"
-      Roles:
-        - !Ref StatusRetrieverFunctionRole
+      DestinationArn: !Ref CSLSSubscriptionEndpointArn
+      LogGroupName: "/aws/lambda/InvokePrivateAPIGatewayFunction"
+      FilterPattern: "{$.message != \"Sensitive info*\"}"
```

Comment:
I don't think we need this filter pattern here, as we are not adding this prefix in api gateway logs